bazaarvoice/emodb
# Documenation updates for github.io documents

## Github Issue #

None (see below)

## What Are We Doing Here?

The documentation for http://bazaarvoice.github.io/emodb contained several issues such as:

* Spelling and/or grammatical errors
* Bad and/or inconsistent formatting
* Out-of-date examples
* Unclear documentation

This PR is the result of my going through those pages and making any updates I found.  As such the changes (aka, &lt;air-quote&gt;improvements&lt;/air-quote&gt;) are very subjective and should be taken as my opinion only.

## How to Test and Verify

1. Check out this PR.
2. Run jekyll locally using [start_local.sh](https://github.com/bazaarvoice/emodb/blob/master/docs/start_local.sh)
3. Check out the documentation.

## Risk

Low; the only updates are documentation.

### Level 

Low

### Required Testing

None, other than documentation review.

### Risk Summary

The only risks are that new errors were introduced or that it ultimately looks different in the github.io page than it does when rendered locally.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
